terraform/modules/nginx/autoscaling.tf

```diff
@@ -87,7 +87,8 @@ cd /home/ubuntu/provisioning && ansible-playbook -c local -i localhost, nginx_pl
     -e elasticsearch_auth='${var.elasticsearch_auth}' \
     -e app_auth='${var.app_auth}' \
     -e aws_region='${data.aws_region.current.name}' \
-    -e nameserver_ip="$(awk '/nameserver/{ print $2; exit}' /etc/resolv.conf)"
+    -e nameserver_ip="$(awk '/nameserver/{ print $2; exit}' /etc/resolv.conf)" \
+    -e nginx_config='${var.nginx_config}'
```

Comment:
I'd call this something like `mode=live` or `mode=maintenance`. As long as it doesn't clash with any existing variables.